libs/remix-ui/static-analyser/src/lib/ErrorRenderer.tsx

```diff
@@ -0,0 +1,76 @@
+import React, { useState } from 'react' //eslint-disable-line
+
+interface ErrorRendererProps {
+  message: any;
+  opt: any,
+  warningErrors: any
+}
+
+const ErrorRenderer = ({ message, opt }: ErrorRendererProps) => {
+  const [, setError] = useState(
```

Comment:
I don't think this is needed since we don't access the state variables.

Reply:
will be removed, but in future when we need to handle `errors` and not `warning` it will be useful.